go.mod

```diff
@@ -1,21 +1,23 @@
 module sigs.k8s.io/cluster-api-provider-openstack
 
-go 1.21
+go 1.21.6
```

Comment:
v1.22.0 is out by the way

Reply:
ok, I think it'll be nice to do it via another PR though; just in case

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1877